template-test/ci-run.sh

```diff
@@ -11,9 +11,16 @@ VARIANT="${VARIANT:-basic}"
 APP_NAME="${VARIANT}-test-app"
 
 rm -rf template-test/dummy/$APP_NAME
+mkdir -p template-test/dummy/$APP_NAME
+
+# otherwise rails webpacker:install initiates yarn init and that overrides these yarn.lock package.json files 
+# exist under the root directory instead of creating new ones under template-test/dummy/$APP_NAME
+cp yarn.lock package.json template-test/dummy/$APP_NAME
```

Comment:
Hi @G-Rath 

I have finally figured out why the CI is failing. 

When we run `template-test/ci-run.sh` if there is a yarn.lock and package.json file exist under rails-template folder (the root folder) `yarn init` overrides these files (i.e. adds newer versions of packages into the file etc… ) instead of creating new ones under `template-test/dummy/basic-test-app` folder where we temporarily create an app to do some tests. 

When I delete these two files the template-test/ci-run.sh  runs without complaining that there is no package.json because webpacker generates these files successfully but ask inputs from the command line when running ci-run script which is not ideal.

I am not sure why we have these files in the root folder of rails-template so now I am copying them inside the ci-run.sh scrip that seems to work ok
i.e. 
cp yarn.lock package.json template-test/dummy/$APP_NAME

I was wondering if you have any other suggestions?

Reply:
@emre-simtay we definitely shouldn't be copying the `yarn.lock` because we want to always try to be using the latest of everything when we generate a project; similarly that's also why we shouldn't copy the `package.json`.

`yarn init` should solve this - I suspect you just need to make sure you're in the right directory, so try something like `cd template-test/dummy/$APP_NAME && yarn init --yes`.

Reply:
Awesome thanks! I did add this run "yarn init" to front end base template but it didn't work at all.  Not sure why maybe I didn't add --yes? 

It's better now, thanks.